lib/jekyll-docs.rb

```diff
@@ -20,17 +18,13 @@ def init_with_program(prog)
       end
 
       def process(opts)
-        Dir.mktmpdir do |dest_dir|
-          options = opts.merge({
-            "serving"     => true,
-            "watch"       => false,
-            "config"      => File.expand_path("../../site/_config.yml", __FILE__),
-            "source"      => File.expand_path("../../site", __FILE__),
-            "destination" => dest_dir
-          })
-          Jekyll::Commands::Build.process(options)
-          Jekyll::Commands::Serve.process(options)
-        end
+        options = opts.merge({
+          "serving"     => true,
+          "watch"       => false,
+          "destination" => File.expand_path("../../site", __FILE__),
```

Comment:
No, it doesn't. Just pass some options to the WEBRick server, like the port and SSL config, but no more than that. Why should Jekyll look for a `_config.yml` when serving? By the way, `_config.yml` is needed when making a instance of `Jekyll::Site` for the `build` command, but `serve` doesn't uses it at all, or at least in Jekyll 3.1.6.